README.md

```diff
@@ -34,20 +38,18 @@ https://github.com/udacity/CarND-Capstone/issues/147 We locally updated our requ
 * Real world testing (will be updated)
 See below
 
-* Suggestions
-I wish Virtual Workspace had internet access so that instead of uploading files it would be possible to clone a repo.
 
 ### Other Issues we ran into
 1. Requirements.txt was modified to point pillow to version 4.3 (see above).  We had an issue with cv.Bridge that was traced back to https://github.com/udacity/CarND-Capstone/issues/147.  This required upgrade to pillow version 4.3
-- We do want to call out that while you mention not to change the requirements.txt, you have used different kinds of settings yourself in the Virtual Workspace
-- This has caused a lot of confusion and wasted time
+  1. We do want to call out that while you mention not to change the requirements.txt, you have used different kinds of settings yourself in the Virtual Workspace
+  2. This has caused a lot of confusion and wasted time
```

Comment:
add another issue. "Latency between ROS & simulator when camera is turned on. This issue has been reported multiple times, but seems to be unaddressed by Udacity. The issue is described in https://github.com/udacity/CarND-Capstone/issues/266 . The issue affects both, provided Workspace and Docker environment. It did not seem to affect the virtual machine environment. It did not affect native installation."